v0/destinations/one_signal/config.js

```diff
@@ -0,0 +1,29 @@
+const { getMappingConfig } = require("../../util");
+
+const BASE_URL = "https://onesignal.com/api/v1";
+
+const ENDPOINTS = {
+  IDENTIFY: {
+    endpoint: "/players"
+  },
+  TRACK: {
+    endpoint: "/apps"
+  },
+  GROUP: {
+    endpoint: "/players"
```

Comment:
For group calls, we can use the `apps` endpoint as we update the existing user with a tag.

Reply:
updated